pkg/controllers/externalsecret/esmetrics/esmetrics.go

```diff
@@ -32,78 +31,37 @@ const (
 	ExternalSecretReconcileDurationKey = "reconcile_duration"
 )
 
-var (
```

Comment:
I moved some of the variables and functions under pkg/controllers/metrics package so that they can be used from the other controllers